Gordon360/Models/ViewModels/RecIM/ParticipantViewModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Gordon360.Models.CCT;
+using Microsoft.Graph;
+using System;
+using System.Collections.Generic;
+
+namespace Gordon360.Models.ViewModels.RecIM
+{
+    public class ParticipantViewModel
+    {
+        public int ID { get; set; }
```

Comment:
I think that Gordon ID should not be included in your ParticipantViewModel unless you have a specific, unavoidable need to send this data to the frontend. This is considered sensitive info and should almost never be visible to users.

Reply:
@amos-cha I think 'ADusername` may be a better replacement of `ID`.

Reply:
@EjPlatzer, I somehow forgot that Gordon ID was redundant for the viewmodel as I already included the unique identifier of ADUserName

@Kevin-Chu58, ADUserName is already included